README.md

```diff
@@ -28,10 +59,19 @@ This repo contains 2 types of issue templates:
 
 This special repository also contains a special file `./profile/README.md` that will appear on the organizations profile.
 
+## Contributing
+
+To contribute to this repository, please see the [contribution guidelines](CONTRIBUTING.md).
+
 <!-- reference urls -->
 
-[gh-action-cvs]: github.com/plantemoran-ai-engineering/.github/actions/workflows/code-analysis.yml
-[gh-action-lint]: github.com/plantemoran-ai-engineering/.github/actions/workflows/linting.yml
+[developer-workstation-config-docs]: https://plantemoran-appdev.github.io/github-process-docs/workstation-setup/workstation-config/
+[gh-action--backup-repo]: https://github.com/plantemoran-ai-engineering/.github/actions/workflows/backup-repo.yml
+[gh-action-cvs]: https://github.com/plantemoran-ai-engineering/.github/actions/workflows/code-analysis.yml
+[gh-action-lint]: https://github.com/plantemoran-ai-engineering/.github/actions/workflows/linting.yml
 [gh-creating-default-comm-health-file]: https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file
-[img-gh-action-cvs-badge]: github.com/plantemoran-ai-engineering/.github/actions/workflows/code-analysis.yml/badge.svg
-[img-gh-action-lint-badge]: github.com/plantemoran-ai-engineering/.github/actions/workflows/linting.yml/badge.svg
+[gh-action--appscan-sast]: https://github.com/plantemoran-ai-engineering/.github/actions/workflows/hcl-appscan-sast-scan.yml
+[img--gh-action-badge--backup-repo]: https://github.com/plantemoran-ai-engineering/.github/actions/workflows/backup-repo.yml/badge.svg
+[img--gh-action-badge--appscan-sast]: https://github.com/plantemoran-ai-engineering/.github/actions/workflows/hcl-appscan-sast-scan.yml/badge.svg
+[img-gh-action-cvs-badge]: https://github.com/plantemoran-ai-engineering/.github/actions/workflows/code-analysis.yml/badge.svg
+[img-gh-action-lint-badge]: https://github.com/plantemoran-ai-engineering/.github/actions/workflows/linting.yml/badge.svg
```

Comment:
Same w/ your badges... stay consistent

Here is how I am currently doing it:
```text
[img--gh-action-badge--appscan-codesweep]
[img--gh-action-badge--appscan-sast]
[img--gh-action-badge--codeql]
[img--gh-action-badge--linting]
etc.
```

Also note how the suffix matches exactly w/ the suffix of the `gh-action--` ones above.

Reply:
Good catch! Fixed.